homeassistant/components/route53.py

```diff
@@ -0,0 +1,133 @@
+"""
+Update the IP addresses of your Route53 DNS records.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/route53/
+"""
+
+from datetime import timedelta
+import logging
+import voluptuous as vol
+from homeassistant.const import CONF_DOMAIN, CONF_ZONE
+import homeassistant.helpers.config_validation as cv
+from homeassistant.helpers.event import track_time_interval
```

Comment:
Please keep the import [grouped](https://developers.home-assistant.io/docs/en/development_guidelines.html).

Reply:
Done.